exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_fix_version.ipynb

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
<p>Can you split this code cell in two parts and add markdown cells for each part explaining the code?</p><p> </p><p>In the first part we create a UDF that reads the version of the scikit_learn library. The UDF runs inside the language container, therefore the library version detected by the UDF is the version installed in this container.</p><p> </p><p>In the second part we compare the version returned by the UDF with the version in the AI-Lab environment. If they differ we install the UDF's version in the AI-Lab environment.</p><p> </p><p>BTW, do we need to run pip with the --upgrade option?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/316/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Sure, see next push.

Reply:
> BTW, do we need to run pip with the --upgrade option?

That I cannot answer. My tests have been successful without this option.

Reply:
<p>Looks good to me</p>